src/reducers/crossSliceReducer.js

```diff
@@ -18,13 +18,14 @@ export default function crossSliceReducer(state = {}, action) {
           c => c.storeAs || c.collection,
         );
 
-        for (const storeAs in groups) {
+        keys(groups).forEach(storeAs => {
```

Comment:
Feels like we might be able to make this a function that returns instead of using `forEach` and `set` on an object from a shared scope.

Either way, nice work!